schemas/json/theme.json

```diff
@@ -231,6 +231,16 @@
 							"description": "Allow users to set text colors.",
 							"type": "boolean",
 							"default": true
+						},
+						"heading": {
+							"description": "Allow users to set heading colors.",
```

Comment:
I find this confusing because it can easily be mixed up with styles > elements > headings.

Reply:
Indeed. What about "Allow users to set heading colors **in a block**."?